packages/core-web/src/styles/scroll-top-button.css

```diff
@@ -0,0 +1,13 @@
+#scroll-top-button {
+    display: none;
+    position: fixed;
+    bottom: 20px;
+    right: 30px;
+    z-index: 99;
+    cursor: pointer;
+    opacity: 40%;
+}
+
+#scroll-top-button:hover {
+    opacity: 70%;
```

Comment:
Alright! Do you have any recommendation for the styles? 

I'm thinking of just a "css default dark-grey color" for the :active state. What do you think?

Reply:
I've added an "opacity: 100%" for this so that when users click on it, they can get a feedback that the button is clicked! The opacity will transition from 40% to 100% from the moment they click, until the moment they release their mouse left click, then it will transition back to 40%.

Reply:
per the above, this is not working as well =C

Reply:
This was resolved by changing percentage values to its represented decimal values!